single-cell-ebi-gxa.yaml

```diff
@@ -6,151 +6,247 @@ install_tool_dependencies: false
+    tool_panel_section_id: 'hca_sc_get-scrna'
     
   - name: hca_matrix_downloader
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_get-scrna'
     
   - name: sc3_calc_biology
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_sc3_tools'
     
   - name: sc3_calc_consens
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_sc3_tools'
     
   - name: sc3_calc_dists
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_sc3_tools'
     
   - name: sc3_calc_transfs
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_sc3_tools'
 #    
   - name: sc3_estimate_k
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_sc3_tools'
     
   - name: sc3_kmeans
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_sc3_tools'
     
   - name: sc3_prepare
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_sc3_tools'
     
   - name: scanpy_compute_graph
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
     
   - name: scanpy_filter_cells
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
     
   - name: scanpy_filter_genes
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
     
   - name: scanpy_find_cluster
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
     
   - name: scanpy_find_markers
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
     
   - name: scanpy_find_variable_genes
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
     
   - name: scanpy_normalise_data
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
     
   - name: scanpy_parameter_iterator
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
+       
+  - name: scanpy_plot_embed
+    owner: ebi-gxa
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
+    
+  - name: scanpy_plot_trajectory
+    owner: ebi-gxa
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
     
   - name: scanpy_read_10x
     owner: ebi-gxa
-    tool_panel_section_label: 'HCA-Single Cell'
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
+        
+  - name: scanpy_regress_variable
+    owner: ebi-gxa
+    tool_panel_section_id: 'hca_sc_scanpy_tools'
```

Comment:
can you please change all `_` for `-` ... I guess we are going with `-` ... disclaimer, this can change, but for now lets try that.

Reply:
wait ... do not change it now, I need to double-check there is something strange going on ...

Reply:
`_` works well on our local instances, or is there another reason?

Reply:
(our local instances are tip of 19.05 release branch)

Reply:
What I noticed though is that the sections only get rendered in the tool panel when there are tools within that section.

Reply:
usegalaxy.* coordination, nothing technical more community agreements ...

Reply:
Do we need to change this then? Let me know